alarmserver/alarmserver@.service

```diff
@@ -0,0 +1,53 @@
+; Before you start
+; ----------------
+;   Replace "{user}" with a real user name when you follow this instruction.
+;   For example, replace it with "pi" without quotes for Raspberry Pi users.
+;
+; Pre-requisites
+; --------------
+;  - "screen" package is needed for this systemd unit
+;
+;      sudo apt-get install screen
+;
+;  - Python "requests" module is needed for Alarmserver python script.
+;  
+;      sudo apt-get install python-pip
+;      pip install requests
+;
+; Service Installation
+; --------------------
+; 1. Save your alarmserver scripts at /home/{user}/alarmserver
+; 2. Save this file at /usr/lib/systemd/system/alarmserver@.service
```

Comment:
should we be advising this location, or rather /etc/systemd/system? I believe the /usr/lib/systemd location is primarily used for packages as opposed to locally/manually installed files like this.

Reply:
How about /usr/lib/alarmserver instead? I found many other software installed are there and they create needed symlink or hardlink to /usr/bin/ if then want the executable available for user to execute anywhere.

/etc is where people store system configuration files but not executables or python scripts.

Reply:
I'm referring specifically to the .service file location - there's really only 2 valid locations for systemd - and that's /usr/lib/systemd/system (which is typically where RPM/Debs drop files) and /etc/systemd/system which is typically used for manually installed files (similar to /usr vs /usr/local, despite the slightly different context of /usr/lib vs /etc)